test/Hamiltonians.jl

```diff
@@ -1880,3 +1880,12 @@ end
     @test LOStructure(h) isa IsDiagonal
     @test_throws ArgumentError adjoint(h2)
 end
+
+@testset "Comparison of ExtendedHubbardMom1D with ExtendedHubbardReal1D" begin
+    addr = FermiFS{3,6}(0,1,1,1,0,0)
+    for boundary_condition in ([i*π for i in 0.0:0.2:1.0])
+        HR = ExtendedHubbardMom1D(addr; boundary_condition)
+        HM = ExtendedHubbardReal1D(addr; boundary_conition= boundary_condition/6)
```

Comment:
```suggestion
        HM = ExtendedHubbardMom1D(addr; boundary_condition)
        HR = ExtendedHubbardReal1D(addr; boundary_conition=boundary_condition/6)
```
Why does the real space version need `boundary_condition / 6`? I think it should be the same for both Hamiltonians. It would also be good to repeat the test with `BoseFS`.